include/tvm/relay/attrs/vision.h

```diff
@@ -60,16 +60,16 @@ struct MultiBoxTransformLocAttrs
 
 /*! \brief Attributes used in non_maximum_suppression operators */
 struct NMSAttrs : public tvm::AttrsNode<NMSAttrs>{
-  double overlap_threshold;
+  double nms_threshold;
   bool force_suppress;
-  int topk;
+  int nms_topk;
 
   TVM_DECLARE_ATTRS(NMSAttrs, "relay.attrs.NMSAttrs") {
-      TVM_ATTR_FIELD(overlap_threshold).set_default(0.5)
+      TVM_ATTR_FIELD(nms_threshold).set_default(0.5)
```

Comment:
We should keep this argument name as overlap_threshold, since we don't need nms prefix. It can generate some inconsistency with current topi nms operator, but topi nms argument names will be modified in SSD refactor PR.